pymemcache/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '3.1.0'
+__version__ = '3.1.1'
```

Comment:
Please remove this part of the change. We'll bump the version as part of the release process.